Jenkinsfile

```
@@ -25,7 +25,7 @@
 // Then a second PR submitted to comment out the @Library line, and when it
 // is landed, both PR branches can be deleted.
 
-@Library(value="pipeline-lib@corci-768a") _
+//@Library(value="pipeline-lib@corci-768a") _
```

```suggestion
//@Library(value="pipeline-lib@your_branch") _
```

To get back to it's original value and a value that is consistent with every other `Jenkinsfile`.